### PR TITLE
Call dev

### PR DESCRIPTION
**Pull request by lzan13**
update video and voice call ui

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
     package="com.hyphenate.chatuidemo"
     >
 
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
```
Comment (githubzhaoliang) [inline app/src/main/AndroidManifest.xml:6]: what case will use bluetooth?

File: app/src/main/java/com/hyphenate/chatuidemo/call/CallPushProvider.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * Send the push message when offline
+ */
+public class CallPushProvider implements EMCallManager.EMCallPushProvider {
+    @Override public void onRemoteOffline(final String username) {
+        final EMMessage message = EMMessage.createTxtSendMessage("offline call", username);
+        message.setAttribute(Constant.MESSAGE_ATTR_IS_CALL_PUSH, true);
+        // 设置强制推送
+        message.setAttribute("em_force_notification", "true");
+        // 设置自定义推送提示
```
Comment (githubzhaoliang) [inline app/src/main/java/com/hyphenate/chatuidemo/call/CallPushProvider.java:22]: change chinese  comments to english comments

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (lzan13) [reply app/src/main/AndroidManifest.xml:6]: 在通话时如果设备连接了蓝牙，需要检测蓝牙设备，然后手动切换到蓝牙输出才能切换蓝牙输出音视频通话的声音，否则不能使用蓝牙

File: app/src/main/java/com/hyphenate/chatuidemo/call/CallPushProvider.java (hunk 2 above)
Comment (lzan13) [reply app/src/main/java/com/hyphenate/chatuidemo/call/CallPushProvider.java:22]: 这里稍等我改下

File: app/src/main/java/com/hyphenate/chatuidemo/call/FloatWindow.java [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                break;
+            case VOICE_RESUME:
+                Toast.makeText(context, "Remote resume voice", Toast.LENGTH_SHORT).show();
+                break;
+            default:
+                break;
+        }
+    }
+
+    private void refreshCallTime() {
```
Comment (githubzhaoliang) [inline app/src/main/java/com/hyphenate/chatuidemo/call/FloatWindow.java:290]: there is similar method in class DateUtils, please consider to reuse them or put this method to that class.

File: app/src/main/java/com/hyphenate/chatuidemo/call/VideoCallActivity.java [hunk 4]
```diff
@@ -1,838 +1,574 @@
...
+        }
     }
 
     /**
-     * stop refresh call info
+     * Refresh call time
      */
-    private void stopMonitor() {
-        mMonitor = false;
+    private void refreshCallTime() {
```
Comment (githubzhaoliang) [inline app/src/main/java/com/hyphenate/chatuidemo/call/VideoCallActivity.java:465]: duplicate code

File: app/src/main/java/com/hyphenate/chatuidemo/call/VoiceCallActivity.java [hunk 5]
```diff
@@ -23,535 +20,353 @@
...
+     * refresh call time
      */
-    private void closeSpeaker() {
-        // Set button state
-        mSpeakerSwitch.setActivated(false);
-        CallStatus.getInstance().setSpeaker(false);
-        if (mAudioManager.isSpeakerphoneOn()) {
-            // Close speaker
-            mAudioManager.setSpeakerphoneOn(false);
+    private void refreshCallTime() {
```
Comment (githubzhaoliang) [inline app/src/main/java/com/hyphenate/chatuidemo/call/VoiceCallActivity.java:282]: duplicate code

**Pull request closed without merging**